10_install_server/12_install_root_server_rhel.txt

```diff
@@ -0,0 +1,99 @@
+=== Install Rudder Root server on Enterprise Linux ( RedHat or RedHat-like )
+
+==== Add the Rudder packages repository
+
+Run the following command as root:
+
+Note: <version> is your RHEL version
```

Comment:
I'm not really keen on duplicating entries endless times (one for each supported OS, doomed to grow over time unless we forget to update the doc which would be worse), so I'd go for your second idea: using RHEL_7 by default to be future proof and adding a note for users to change if necessary. (RHEL_7 is the current released OS from Red Hat,)